CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Change Log
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and adheres to [Semantic Versioning](http://semver.org) as of version 1.0.0.
```

Comment:
Okay, SemVer it is, I'll submit a PR to define the API in the README (just dump the current one).